### PR TITLE
Improve check processing logs

### PR DESCRIPTION
**Pull request by danfaizer**

### CODE REVIEW

File: jobrunner/job.go [hunk 1]
```diff
@@ -20,4 +21,20 @@ type Job struct {
...
+	return fmt.Sprintf(
+		"event=checkTrace context=\"%s\" action=%s checkID=%s target=%s assetType=%s checkImage=%s createTime=\"%s\" queuedTime=%d runningTime=%d",
+		context,
+		action,
+		j.CheckID,
+		j.Target,
+		j.AssetType,
+		j.Image,
+		j.StartTime,
+		j.RunTime-j.StartTime.Unix(),
```
Comment (jesusfcr) [inline jobrunner/job.go:37]: this produces `2022-11-25 15:44:47.555215432 +0000 UTC`. IMHO it's too much accurate/verbose.

File: jobrunner/job.go [hunk 2]
```diff
@@ -20,4 +21,20 @@ type Job struct {
 	Options      string            `json:"options"`       // Optional
 	RequiredVars []string          `json:"required_vars"` // Optional
 	Metadata     map[string]string `json:"metadata"`      // Optional
+	RunTime      int64
+}
+
+func (j *Job) logTrace(context, action string) string {
+	return fmt.Sprintf(
+		"event=checkTrace context=\"%s\" action=%s checkID=%s target=%s assetType=%s checkImage=%s createTime=\"%s\" queuedTime=%d runningTime=%d",
```
Comment (jesusfcr) [inline jobrunner/job.go:29]: what about `s/contest/msg/`

File: jobrunner/job.go (hunk 1 above)
Comment (jesusfcr) [reply jobrunner/job.go:37]: Also, is it useful? it could be removed, or just replace with    now-startTime.

File: jobrunner/job.go (hunk 2 above)
Comment (danfaizer) [reply jobrunner/job.go]: Agree. Done here [63913f3](https://github.com/adevinta/vulcan-agent/pull/50/commits/63913f3740a63d35866fe8b915c49e00e4d9be9a)

File: jobrunner/job.go (hunk 1 above)
Comment (danfaizer) [reply jobrunner/job.go:37]: Agree. Removed here [03e280b](https://github.com/adevinta/vulcan-agent/pull/50/commits/03e280ba0ddb76c668c306c8f87cb273b601df14)

**Review by jesusfcr — APPROVED** [review]
Nice!

**Pull request merged**